src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml

```diff
@@ -56,7 +56,14 @@
     </div>
     <footer class="footer border-top pl-3 text-muted">
         <div class="container">
-        &copy; @DateTime.Now.Year - @Environment.ApplicationName - <a asp-area="" asp-page="Privacy">Privacy</a>
+        &copy; @DateTime.Now.Year - @Environment.ApplicationName -
+            @{
```

Comment:
I find it odd to have a runtime check for this file in here. That's certainly not what any real app would do. Maybe it's best to omit the link entirely, and then it's up to the app to override the layout file to put in the privacy link if they want it?

Reply:
@blowdart are you ok if we just omit the privacy link from the identity UI login page?  Its already only there in V4 apps.  So removing it just gets us back to the V3 situation anyways.

The privacy link is still in the main index page, it just would be omitted from identity UI pages

Reply:
No. This has GDPR implications. The privacy link can't just vanish from pages, especially pages that take PII. It needs to be everywhere.

Reply:
Was just expressing my opinion. Ultimately the Identity UI is kind of a weird case of pre-packaged UI, so there aren't really established rules for it. Blaze your own path :)

Reply:
I think it’s fine. This doesn’t show up anywhere else but in our implementation.

We do a similar thing to detect login partial and throw a specific error if missing.